internal/decodejpeg/lib/pkgconfig/libjpeg.pc

```diff
@@ -0,0 +1,10 @@
+prefix=/home/joakim/dev-12bit
```

Comment:
This doesn't look intentional.

Reply:
that is info about how it is built - I can remove the pkgconfig dir, but I thought it would be good info.

Reply:
OK, removed